Classes/BEMSimpleLineGraphView.m

```diff
@@ -136,6 +136,10 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
     return self;
 }
 
+-(void)awakeFromNib{
```

Comment:
Why is this necessary? `commonInit` will now be called twice, which is not good.